app/subscriber/src/store/slices/profile/interfaces/IProfileState.ts

```diff
@@ -14,4 +15,5 @@ export interface IProfileState {
   myMinisters: IMinisterModel[];
   contributors: IContributorModel[];
   systemMessages: ISystemMessageModel[];
+  sources: ISourceModel[];
```

Comment:
Why are you storing these here?  The `lookupSlice` has a place for sources.

Reply:
The reason I need it from here is because the productId is not contained on the lookups and I need them to separate the sources in the media filter

Reply:
I don't understand why you are storing them in the user profile store.  Sources is a list of lookups.  Keep the implementation consistent or it's more difficult to maintain.  We have a sources list in redux store already in the lookups slice.

Reply:
Ohh I had a brainfart, for some reason I thought using the lookup store we had to conform to a certain interface that all lookups matched.. but a lot of the products just don't have productIds 🤦‍♂️ - changing